tools/devops/automation/templates/sign-and-notarized/upload-azure.yml

```diff
@@ -32,6 +32,9 @@ steps:
     persistCredentials: true
     path: s/xamarin-macios
 
+- checkout: yaml-templates
```

Comment:
A comment explaining why we're checking out yaml-templates might be useful.

Reply:
yes, cause I bet someone will delete it in the future.
```suggestion
# checkout an extra repo to ensure that we have the same tree structure in the working directory in all pipelines.
# if you delete this checkout the unified pipeline will have issues.
- checkout: yaml-templates
```